Makefile

```diff
@@ -135,6 +135,11 @@ jvminstall:
 		mvn install -P$(JVM_PKG_PROFILE) -Dcxx="$(CXX)" \
 			-Dcflags="$(PKG_CFLAGS)" -Dldflags="$(PKG_LDFLAGS)" \
 			-Dcurrent_libdir="$(ROOTDIR)/$(OUTPUTDIR)" $(JVM_TEST_ARGS))
+format:
+	./tests/lint/git-clang-format.sh -i origin/master
```

Comment:
This should probably use `tests/lint/git-black.sh` and `tests/lint/git-clang-format.sh`

Reply:
So I think running Black without the caching is actually more aggressive/safe, but YMMV, happy to change it.